src/components/ProductBacklogListComponent.js

```diff
@@ -307,30 +383,51 @@ const ProductBacklogListComponent = () => {
                   color="primary"
                   variant="contained"
                   style={{ flex: 1, width: '40%' }}
-                  onClick={() => {
+                  onClick={async () => {
                     //Make sure the story is valid first
```

Comment:
ideally move this block to it's own named function to not bloat the JSX too much, nitpick though